level2/[level2] 정윤주/application/src/main/java/org/example/Application.java

```diff
@@ -0,0 +1,60 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Application {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("덧셈할 문자열을 입력해 주세요.");
+        String str = scanner.nextLine();
+
+        System.out.println("결과 : ");
+
+        scanner.close();
+
+    }
+
+    //기본 구분자로 문자열 분리
+    public static ArrayList<Integer> splitBasic(String str) {
```

Comment:
```suggestion
    public static ArrayList<Integer> splitBasic(String inputValue) {
```
좀 더 의미있는 변수명을 사용하면 좋을 것 같습니다.